### PR TITLE
Animation added for HomePage

### PR DESCRIPTION
**Pull request by dev04sa**
@pooranjoyb as required Animations added to HomePage by using Framer-motion .
Issue https://github.com/pooranjoyb/popShop/issues/38 also Resolved .
Kindly Review The PR.

### CODE REVIEW

File: src/pages/Home/Home.tsx [hunk 1]
```diff
@@ -32,13 +60,17 @@ function Home() {
             </div>
           </div>
         </div>
-        <img className="w-2/3 sm:w-1/3 py-4 sm:p-0" src="./images/hero.png" alt="" />
```
Comment (Mahitej28) [inline src/pages/Home/Home.tsx:35]: @dev04sa kindly don't change the responsive of the website

**Review by Mahitej28 — CHANGES_REQUESTED** [review]
Don't Change the repsonsiveness of website like don't delete the sm and max-screen values.
@dev04sa

**Comment by Mahitej28** [comment]
@dev04sa why are this images coming small as compared to others kindly look into this.
image

**Comment by dev04sa** [comment]
@Mahitej28  , I will look into changes.

**Comment by Mahitej28** [comment]
Looks good to me!

**Pull request merged**